src/core_atmosphere/mpas_atm_core.F

```diff
@@ -862,15 +862,55 @@ subroutine atm_do_timestep(domain, dt, itimestep)
       xtime_s = (s + s_n / s_d)
 
 
+      !
+      ! This is a call to all of the slow, column-oriented physics, except for resolved
+      ! cloud microphysical processes. The microphysics is handled after the dynamics so 
+      ! that any supersaturation caused by various dynamical processes (advection, etc) 
+      ! may be removed before the end of each time step.
+      !
 #ifdef DO_PHYSICS
       !proceed with physics if moist_physics is set to true:
       if(moist_physics) then
```

Comment:
Would there ever be a situation where radiation (for example) would be called, but not microphysics?